examples/intellij-plugin/src/main/kotlin/com/jetbrains/compose/color/ColorLineMarkerProvider.kt

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
Could you move this example into 'intellij-plugin-with-experimental-shared-base' from 'intelij-plugin'? `intellij-plugin-with-experimental-shared-base` will be the recommended way to develop plugins for IDEA. Shared base bundles Compose, and other plugins use it. Without this shared base, every Compose plugin would contain its own version of Compose.